docs/src/main/paradox/developer/index.md

```diff
@@ -2,7 +2,7 @@
 
 This section covers information for developers of Cloudstate itself.
 
-There is a weekly Cloudstate contributors' call, see the following for [details](https://calendar.google.com/event?action=TEMPLATE&tmeid=MXNmaXA4Y3NndjBrbm1wN2N2aWNja2ZjdW1fMjAxOTA5MTdUMjAwMDAwWiBsaWdodGJlbmQuY29tX3RrMDF0cDF0MjRjaGwwODlqZmZhaXQwdGM4QGc&tmsrc=lightbend.com_tk01tp1t24chl089jffait0tc8%40group.calendar.google.com&scp=ALL).
+There are two weekly Cloudstate contributors' calls, which are scheduled so that people from around the world can attend. The schedule of calls can be found in [this](https://calendar.google.com/calendar/embed?src=lightbend.com_3is0shfmmqgh4q57eo0n8m6bfc%40group.calendar.google.com) callendar. You can add this calendar to your Google calendar by clicking [here](https://calendar.google.com/calendar/b/1?cid=bGlnaHRiZW5kLmNvbV8zaXMwc2hmbW1xZ2g0cTU3ZW8wbjhtNmJmY0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t), and you can import this into calendar apps that support iCal using [this](https://calendar.google.com/calendar/ical/lightbend.com_3is0shfmmqgh4q57eo0n8m6bfc%40group.calendar.google.com/public/basic.ics) link.
```

Comment:
typo: `callendar`

```suggestion
There are two weekly Cloudstate contributors' calls, which are scheduled so that people from around the world can attend. The schedule of calls can be found in [this](https://calendar.google.com/calendar/embed?src=lightbend.com_3is0shfmmqgh4q57eo0n8m6bfc%40group.calendar.google.com) calendar. You can add this calendar to your Google calendar by clicking [here](https://calendar.google.com/calendar/b/1?cid=bGlnaHRiZW5kLmNvbV8zaXMwc2hmbW1xZ2g0cTU3ZW8wbjhtNmJmY0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t), and you can import this into calendar apps that support iCal using [this](https://calendar.google.com/calendar/ical/lightbend.com_3is0shfmmqgh4q57eo0n8m6bfc%40group.calendar.google.com/public/basic.ics) link.
```